tests/js/spec/stores/discoverSavedQueriesStore.spec.jsx

```diff
@@ -11,9 +11,9 @@ describe('DiscoverSavedQueriesStore', function() {
   let api;
   const now = '2019-09-03T12:13:14';
 
-  beforeAll(function() {
+  beforeAll(async function() {
     api = new Client();
-    DiscoverSavedQueriesStore.reset();
+    await DiscoverSavedQueriesStore.reset();
```

Comment:
@markstory Is this really an async function? Seems to me that if you put `await tick()` after, that would work as well.

Reply:
Yeah that should work. I'll do that instead.